### ISSUE TITLE
Request for useful error message on "type:string"

### ISSUE THREAD

**Issue by RubenGarcia** [go-swagger/go-swagger#2376]
## Problem statement

if a swagger is missing a space in "type: string" and is written "type:string"
swagger generate server produces "json: cannot unmarshal string into Go struct field SwaggerProps.definitions of type struct { spec.SchemaProps; spec.SwaggerSchemaProps }"
Something like
"Missing space in 'type:string' would be a better error message

## Swagger specification
---
swagger: "2.0"
basePath: "/api/v0.1.2"
info:
  title: Test
  description: Test
  version: "0.1"
  contact:
    email: a@b.example.com
  license:
    name: Apache 2.0
    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'

definitions:
  Eudat:
    type: object
    properties:
      EUDATFIO:
        type:string 

paths:
  /test:
    get:
      responses:
        '200':
          description: test
          schema:
            $ref: "#/definitions/Eudat"

## Steps to reproduce
 swagger generate server

## Environment
swagger version: v0.25.0
go version: 1.14.2
OS: Ubuntu 18.04.4 LTS

**Comment by fredbi** [comment]
Yes, the validator first unmarshals the spec and there is no attempt to keep the full context of the unmarshaling error.
This is another instance of old issues #1425 and #1308. Basically, this was the intent with #121, and we never completed it.

This kind of improvement (most likely in go-openapi/validate) would be indeed welcome.
